File: user-and-ownership-service/src/main/java/com/example/EV_Co_ownership/User_and_ownership_service/DTO/UserDTO.java
package com.example.EV_Co_ownership.User_and_ownership_service.DTO;

import lombok.Getter;
import lombok.Setter;

import java.time.LocalDateTime;


@Setter
@Getter
public class UserDTO {
    private int userId;
    private String email;
    private int role_id;
    private String password;
    private boolean isVerified;
    private boolean isDeleted;
    private LocalDateTime createdAt;
}

File: admin-and-staff-service/src/modules/common/user.dto.ts
export class UserResponseDto {
  user_id: number;
  email: string;
  isVerified: boolean;
  isDeleted: boolean;
  createdAt: Date;
  role: string; // hoặc object nếu bạn muốn trả chi tiết role
  passwordResetToken?: string | null;
  resetTokenExpiry?: Date | null;
}
